frontend/src/locales/en.json

```diff
@@ -1,355 +1,358 @@
 {
     "UNSAFE ALERT - UnsafeAlert.js": "Alert that shows on top when browsing from the unsafe clearnet sites",
     "You are not using RoboSats privately": "You are not using RoboSats privately",
-    "desktop_unsafe_alert": "Some features are disabled for your protection (e.g. chat) and you will not be able to complete a trade without them. To protect your privacy and fully enable RoboSats, use <1>Tor Browser</1> and visit the <3>Onion</3> site.",
-    "phone_unsafe_alert": "You will not be able to complete a trade. Use <1>Tor Browser</1> and visit the <3>Onion</3> site.",
-    "Hide":"Hide",
+    "desktop_unsafe_alert": "Some features are disabled for your protection (e.g. chat) and you will not be able to complete a trade without them. To protect your privacy and fully enable RoboSats, use <1>Tor Browser</1> and visit the <3>onion site</3>.",
+    "phone_unsafe_alert": "You will not be able to complete a trade. Use <1>Tor Browser</1> and visit the <3>onion site</3>.",
+    "Hide": "Hide",
 
 
     "USER GENERATION PAGE - UserGenPage.js": "Landing Page and User Generation",
     "Simple and Private LN P2P Exchange": "Simple and Private LN P2P Exchange",
-    "This is your trading avatar":"This is your trading avatar",
-    "Store your token safely":"Store your token safely",
-    "A robot avatar was found, welcome back!":"A robot avatar was found, welcome back!",
```

Comment:
The key of the translation (the sentence on the left) must match perfectly the keys that used in the translate function `t("")`. For example, the key here must match what is inside of `t()` in https://github.com/Reckless-Satoshi/robosats/blob/8d5ea4443abdf772fae9c01aedd99e43d62aacff/frontend/src/components/UserGenPage.js#L175

We cannot simply edit the key of the dictionary in this file without changing every file in https://github.com/Reckless-Satoshi/robosats/tree/main/frontend/src/components . Otherwise the sentence will not be found and the translation will fail.

In addition, if the key is change in the components, it will have to be changed in all: es.json, ru.json... etc.

Reply:
Clearly I'm not an expert... so sorry for that, I can:
a) follow every change in the left side of the en.json file and request also the change on the src file.
b) revert every change in the left side

About the other languages, sure, my plan was request changes when this en.json is done!

Reply:
Haha, yes, I know why you did it this way ^^

The best solution would be (a). We keep the right senstence in English as the key for the translation, editing every instance of the new sentences in the components found in `/frontend/src/components/`. However, this is a lot of work (mostly Ctrl+F, and copy/paste). Otherwise, we can (b) leave the sentences on the left as they were, but this is not a very elegant solution. It is up to you and the time you might have available :)

Reply:
I downloaded a local copy of the repo and I can do the work